test/fixtures/plugins.js

```diff
@@ -0,0 +1,19 @@
+exports.valid = {
+  dependencies: {
+    'rnpm-plugin-link': '1.0.0',
+  },
+  devDependencies: {
+    'rnpm-plugin-build': '1.0.0',
+  },
+};
+
+exports.duplicates = {
+  dependencies: {
+    'rnpm-plugin-link': '1.0.0',
+  },
+  devDependencies: {
+    'rnpm-plugin-link': '1.0.0',
+  },
+};
+
+exports.empty = {};
```

Comment:
Like..? Just asking because I had quite a few issues with going through them :( Anyway, I prefer the above since it aligns with what we are already doing so maybe it's possible to implement your different purpose as well.